tests/tck/utils/utils.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+#
+
+from nebula2.common.ttypes import Value,Vertex,Edge,Path,Step,NullType,DataSet,Row,List
+import nbv
+from behave import model as bh
```

Comment:
maybe we need to consider whether to use pytest-bdd as our backend implementation